dds/DCPS/FilterEvaluator.cpp

```diff
@@ -129,19 +143,45 @@ FilterEvaluator::~FilterEvaluator()
   delete filter_root_;
 }
 
+bool FilterEvaluator::has_non_key_fields(const MetaStruct& meta) const
+{
+  // Check Order
```

Comment:
Is this line (and 158) needed?

Reply:
Probably not, I just try to comment on what think of as "paragraphs" of code and these are definitely short and simple paragraphs so I would be alright with taking them out.